src/main/java/ioc/utils/JsonUtil.java

```diff
@@ -0,0 +1,7 @@
+package ioc.utils;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class JsonUtil {
```

Comment:
Since you are using Spring and it manages all the beans make object mapper a singleton bean instead and use it's only instance where necessary